spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -126,6 +155,48 @@ static void createDatabase(DatabaseAdminClient dbAdminClient, DatabaseId id) {
   }
   // [END spanner_create_database]
 
+  // [START spanner_create_table_with_timestamp_column]
+  static void createTableWithTimestamp(DatabaseAdminClient dbAdminClient, DatabaseId id) {
+    Operation<Void, UpdateDatabaseDdlMetadata> op = dbAdminClient
+        .updateDatabaseDdl(
+            id.getInstanceId().getInstance(),
+            id.getDatabase(),
+            Arrays.asList(
+                "CREATE TABLE Performances (\n"
+                    + "  SingerId     INT64 NOT NULL,\n"
+                    + "  VenueId      INT64 NOT NULL,\n"
+                    + "  EventDate    Date,\n"
+                    + "  Revenue      INT64, \n"
+                    + "  LastUpdateTime TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true)\n"
+                    + ") PRIMARY KEY (SingerId, VenueId, EventDate),\n"
+                    + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE"), null);
+    op.waitFor().getResult();
+    System.out.println("Created Performances table in database: [" + id + "]");
+  }
+  // [END spanner_create_table_with_timestamp_column]
+
+  // [START spanner_insert_data_with_timestamp_column]
+  static void writeExampleDataWithTimestamp(DatabaseClient dbClient) {
+    List<Mutation> mutations = new ArrayList<>();
+    for (Performance performance : PERFORMANCES) {
+      mutations.add(
+          Mutation.newInsertBuilder("Performances")
```

Comment:
Not required, but it would probably be more readable as:
.set("SingerId").to(performance.singerId)
.set("VenueId").to(performance.venueId)
...